tendrl_node_agent/monitoring/constants.py

```diff
@@ -0,0 +1,18 @@
+import platform
```

Comment:
No, this is not required to be exposed via config file. These paths are default paths that the collectd looks at and I don't think the collectd allows to change these values. Yes, the socketgroup and pluginconf file can be changed but, accordingly the collectd's default conf file will then need to change so, if we expose these via conf file, then on every change by the user in the new conf file will then need to be enforced on all nodes and these configs are something that are least likely to be changed by user. I have it here, only because some of the paths will change based on os of the node hosting collectd. the job of which is done by using the capabilities of python's platform lib and w/o the user having to make any choice, internally the choices will be made.